### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by bugagashenkj**

### CODE REVIEW

File: test/applySystemConfig.js [hunk 1]
```diff
@@ -29,63 +29,48 @@ metatests.test('Must properly apply system config', async test => {
...
-            'notAllowedAdditionalProp',
-            'invalidSchema.invalidDomain',
-            {
-              allowed: ['domain'],
-            }
-          ),
-        ])
-      );
-    }
+  await load(validSchemaPath, newConfig.options, newConfig.config);
```
Comment (nechaido) [inline test/applySystemConfig.js]:
```suggestion
  test.resolves(load(validSchemaPath, newConfig.options, newConfig.config));
```

File: test/applySystemConfig.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const {
 const { getSchemaDir } = require('./utils');
 const systemSchemaPath = getSchemaDir('systemSchema');
 
-metatests.test('Must properly apply system config', async test => {
+metatests.test('System config / applying system config', async test => {
```
Comment (nechaido) [inline test/applySystemConfig.js]: I prefer the previous message.
Also, I'd like to use lowercase messages.

File: test/applySystemConfig.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const {
 const { getSchemaDir } = require('./utils');
 const systemSchemaPath = getSchemaDir('systemSchema');
 
-metatests.test('Must properly apply system config', async test => {
+metatests.test('must propaply apply system config', async test => {
```
Comment (lundibundi) [inline test/applySystemConfig.js]:
```suggestion
metatests.test('must properly apply system config', async test => {
```

**Comment by nechaido** [comment]
Tests should always be named `'must do something'`

File: test/applySystemConfig.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ const {
 const { getSchemaDir } = require('./utils');
 const systemSchemaPath = getSchemaDir('systemSchema');
 
-metatests.test('Must properly apply system config', async test => {
+metatests.test('must propeply apply system config', async test => {
```
Comment (belochub) [inline test/applySystemConfig.js]:
```suggestion
metatests.test('must properly apply system config', async test => {
```

File: test/decorators.js [hunk 5]
```diff
@@ -12,7 +12,7 @@ const {
 
 const FLAGS = ['FlagsFromValues', 'FlagsFromEnum', 'FlagsOf'];
 
-metatests.test('Decorators / Flags', async test => {
+metatests.test('must properly creates Flags', async test => {
```
Comment (belochub) [inline test/decorators.js]:
```suggestion
metatests.test('must properly create Flags', async test => {
```

File: test/errors.js [hunk 6]
```diff
@@ -3,7 +3,7 @@
 const metatests = require('metatests');
 const { SchemaValidationError, ValidationError } = require('../lib/errors.js');
 
-const errorsTest = metatests.testSync('errors test');
+const errorsTest = metatests.testSync('must properly throws validation errors');
```
Comment (belochub) [inline test/errors.js]:
```suggestion
const errorsTest = metatests.testSync('must properly throw validation errors');
```

File: test/schema.js [hunk 7]
```diff
@@ -38,11 +30,9 @@ metatests.testAsync('Metaschema create', async test => {
     () => ms.create('category', { definition: {} }, {}),
     new TypeError(`No creator defined for type: 'category'`)
   );
-
-  test.end();
 });
 
-metatests.testAsync('Metaschema addSchemas', async test => {
+metatests.testAsync('ms.addSchemas mut properly add schema', async test => {
```
Comment (belochub) [inline test/schema.js]:
```suggestion
metatests.testAsync('ms.addSchemas must properly add schema', async test => {
```

File: test/default.unit.js [hunk 8]
```diff
@@ -9,51 +9,54 @@ const {
...
-});
+    test.strictSame(config.processOrder, {
+      domains: 0,
+      category: 1,
+    });
+  }
+);
 
-metatests.testSync('default decorators', test => {
+metatests.testSync('must be a full list of default decorators', test => {
```
Comment (nechaido) [inline test/default.unit.js]:
```suggestion
metatests.testSync('must export a full list of default decorators', test => {
```

File: test/errors.js [hunk 9]
```diff
@@ -3,7 +3,7 @@
 const metatests = require('metatests');
 const { SchemaValidationError, ValidationError } = require('../lib/errors.js');
 
-const errorsTest = metatests.testSync('errors test');
+const errorsTest = metatests.testSync('must properly throw validation errors');
```
Comment (nechaido) [inline test/errors.js]:
```suggestion
const errorsTest = metatests.testSync('must correctly serialize validation errors');
```

File: test/default.js [hunk 10]
```diff
@@ -46,27 +46,22 @@ const Person = {
...
-  for (const [name, domain] of ms.domains) {
-    test.strictSame(domain, domains[name]);
-  }
+    for (const [name, category] of ms.categories) {
+      test.strictSame(category, categories[name]);
+    }
 
-  for (const [name, category] of ms.categories) {
-    test.strictSame(category, categories[name]);
+    test.end();
```
Comment (belochub) [inline test/default.js]: I think this can also be removed.

**Pull request merged**